discovery-provider/src/models.py

```diff
@@ -43,14 +43,17 @@ def validate_field_helper(field, value, model):
     return value
 
 def get_default_value(field, value, model, e):
-    schema = ModelValidator.get_schema_for_field(None, model)
     field_props = ModelValidator.get_properties_for_field(model, field)
-    field_type = field_props['type']
+    field_type = field_props['type'] #this can either be a string or list, need to check both
     default_value = field_props['default']
 
     # if the field is of type object and the default value isn't set in the schema, set to SQL null
     # otherwise JSONB columns get set to string 'null'
-    if field_type == 'object' and not default_value:
+    
+    # if field is defined as a list, need to check if 'object' is in list, else check string
```

Comment:
im a little confused by the last part of the comment "else check string". in the else case, it's checking if the field type is object and there's no default value present?

Reply:
so basically the field_type (confusing name) is from the schema and it's either a string or a list. basically if it's a list we need to check that 'object' is a value in the list. else if it's a string, we check if it's a string